src/jiraApi.gs

```diff
@@ -94,7 +100,7 @@ function testConnection() {
  * Performs an request to Jira RESTfull API.
  */
 function Request() {
-  var statusCode, httpResponse, responseData,
+  var statusCode, httpResponse, httpMethod, responseData,
```

Comment:
I had to be able to POST and PUT for the updates in JIRA. The code below is purely to enable that.
GETs should not be affected and I unit tested for that as well as manually testing

Reply:
Agreed - will do.